lwreg/helpers.py

```diff
@@ -6,43 +6,34 @@ def interactive_config():
     """
     print("This is an interactive configuration assisstant for lwreg. \n Please manually add host, user and password to your configuration.")
     config = utils._defaultConfig
+    config.pop("lwregSchema")
```

Comment:
What's this change?

Reply:
lwregSchema is in the utils._defaultConfig and is hence set at the beginning there. However, it doesn't apply to sqlite, that is why I took it out and it will only be added later again if postgresql is the dbtype